variants/CY8CKIT-062S2-AI/pins_arduino.h

```diff
@@ -46,6 +46,10 @@
 #define BUTTON1 14            // Additional BUTTON1
 #define USER_BUTTON BUTTON1   // Standard Arduino USER_BUTTON: Uses BUTTON1
 
+#define GPIO_INTERRUPT_PRIORITY 3 // GPIO interrupt priority
```

Comment:
This couldn´t be directly in the Arduino.h
As  digitalPinToInterrupt() would have the same definition for all boards or?